src/Dfe.PlanTech.Application/Dfe.PlanTech.Application.csproj

```diff
@@ -1,4 +1,10 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <ItemGroup>

```

Comment:
Are these needed? Same with lines 15-17

Reply:
it was weird - that came off the back of merging from main. looks like a folder delete?